src/main/java/org/kitteh/irc/client/library/command/AwayCommand.java

```diff
@@ -65,7 +66,7 @@ public AwayCommand message(@Nonnull String message) {
      * @return this command
      */
     @Nonnull
-    public AwayCommand messageRemove() {
+    public AwayCommand notAway() {
```

Comment:
Yeah, I'd be pretty happy if it was left as `messageRemove` - I don't see it as the same as the ones on the `ClientBuilder` which you've removed.